invenio_stats/ext.py

```diff
@@ -177,6 +178,11 @@ def queries(self):
                 ),
                 permission_factory=cfg.get('permission_factory')
             )
+
+            index = prefix_index(
```

Comment:
Moving this logic in the query class (as e.g. was done for the events and aggregations) would be cleaner